tsl/test/expected/chunk_api-11.out

```diff
@@ -819,8 +821,113 @@ SELECT tablespace FROM pg_tables WHERE tablename = :'CHUNK_NAME';
+-- created via create_chunk_table and having a different name.
+CREATE TABLE devices (id int PRIMARY KEY);
+INSERT INTO devices VALUES (1);
+CREATE TABLE chunkapi (time timestamptz NOT NULL PRIMARY KEY, device int REFERENCES devices(id), temp float  CHECK(temp > 0));
+SELECT * FROM create_hypertable('chunkapi', 'time');
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+            11 | public      | chunkapi   | t
+(1 row)
+
+CREATE TABLE newchunk (time timestamptz NOT NULL, device int, temp float);
+SELECT * FROM test.show_constraints('newchunk');
+ Constraint | Type | Columns | Index | Expr | Deferrable | Deferred | Validated 
+------------+------+---------+-------+------+------------+----------+-----------
+(0 rows)
+
+INSERT INTO newchunk VALUES ('2018-01-01 05:00:00-8', 1, 23.4);
+\set ON_ERROR_STOP 0
+-- Creating the chunk without required CHECK constraints on a table
+-- should fail. Currently, PostgreSQL only enforces presence of CHECK
+-- constraints, but not foreign key, unique, or primary key
+-- constraints. We should probably add checks to enforce the latter
+-- too or auto-create all constraints.
+SELECT * FROM _timescaledb_internal.create_chunk('chunkapi', :'SLICES', :'CHUNK_SCHEMA', :'CHUNK_NAME', 'newchunk');
+ERROR:  child table is missing constraint "chunkapi_temp_check"
+\set ON_ERROR_STOP 1
+-- Add the missing CHECK constraint. Note that the name must be the
+-- same as on the parent table.
+ALTER TABLE newchunk ADD CONSTRAINT chunkapi_temp_check CHECK (temp > 0);
+SELECT * FROM _timescaledb_internal.create_chunk('chunkapi', :'SLICES', :'CHUNK_SCHEMA', :'CHUNK_NAME', 'newchunk');
+ chunk_id | hypertable_id |      schema_name      |     table_name     | relkind |                     slices                     | created 
+----------+---------------+-----------------------+--------------------+---------+------------------------------------------------+---------
+       13 |            11 | _timescaledb_internal | _hyper_10_10_chunk | r       | {"time": [1514419200000000, 1515024000000000]} | t
+(1 row)
+
+-- Show the chunk and that names are what we'd expect
+SELECT
+	:'CHUNK_SCHEMA' AS expected_schema,
+	:'CHUNK_NAME' AS expected_table_name,
+	(_timescaledb_internal.show_chunk(ch)).*
+FROM show_chunks('chunkapi') ch;
+ expected_schema | expected_table_name | chunk_id | hypertable_id | schema_name | table_name | relkind | slices 
+-----------------+---------------------+----------+---------------+-------------+------------+---------+--------
+(0 rows)
+
+-- The chunk should inherit the hypertable
+SELECT relname
+FROM pg_catalog.pg_inherits, pg_class 
+WHERE inhrelid = (:'CHUNK_SCHEMA'||'.'||:'CHUNK_NAME')::regclass AND inhparent = oid;
+ relname  
+----------
+ chunkapi
+(1 row)
+
+-- Test that it is possible to query the data via the hypertable
+SELECT * FROM chunkapi ORDER BY 1,2,3;
+ time | device | temp 
+------+--------+------
```

Comment:
Again, this should show data, right?

Reply:
Agree, looks like this should show previously inserted data into the chunk table

Reply:
Should be fixed now.